homeworks/editerdelyhorvath/2_vars_datatypes/user_info.py

```diff
@@ -0,0 +1,191 @@
+'''
+2. házi feladat
+
+Feladat 2: List és dictionary műveletek használata
+
++ Extra 1 és 2 feladatok (szorgalmi)
+'''
+
+from pprint import pprint
+import os
+
+# Function to clear the terminal screen
+def clear_terminal():
```

Comment:
`import clear_terminal from fictive_character`

így nem kell újra definiálnod ugyanazt itt is. Vagy ami még jobb, csinálhatsz egy külön .py file-t ami tartalmazza ezt a function-t egy leíró névvel. terminal_clearer.py, és mindkét házidba innen importálod be.

Reply:
Ez milyen menő így, köszönöm! Imádom.